src/features/Room/RoomEntryModal/RoomEntryModal.module.scss

```
@@ -86,6 +87,7 @@
 
     span {
       @include m.text-style('sm', 400);
+      color: inherit;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)